src/main/scala/com/github/unchama/seichiassist/subsystems/buildcount/subsystems/notification/System.scala

```diff
@@ -1,28 +1,36 @@
 package com.github.unchama.seichiassist.subsystems.buildcount.subsystems.notification
 
-import cats.effect.ConcurrentEffect
+import cats.effect.{ConcurrentEffect, Sync}
 import com.github.unchama.generic.effect.stream.StreamExtra
-import com.github.unchama.minecraft.actions.OnMinecraftServerThread
 import com.github.unchama.seichiassist.subsystems.buildcount.BuildCountAPI
-import com.github.unchama.seichiassist.subsystems.buildcount.subsystems.notification.application.actions.NotifyLevelUp
-import com.github.unchama.seichiassist.subsystems.buildcount.subsystems.notification.bukkit.actions.BukkitNotifyLevelUp
+import com.github.unchama.seichiassist.subsystems.buildcount.subsystems.notification.application.actions.{NotifyBuildAmountThreshold, NotifyLevelUp}
+import com.github.unchama.seichiassist.subsystems.buildcount.subsystems.notification.bukkit.actions.{BukkitNotifyBuildAmountThreshold, BukkitNotifyLevelUp}
 import com.github.unchama.seichiassist.subsystems.discordnotification.DiscordNotificationAPI
 import io.chrisdavenport.log4cats.ErrorLogger
 import org.bukkit.entity.Player
 
 object System {
 
-  def backgroundProcess[F[
+  def backgroundProcess[F[_]: Sync: ConcurrentEffect: ErrorLogger: DiscordNotificationAPI, G[
```

Comment:
```suggestion
  def backgroundProcess[F[_]: Concurrent: ErrorLogger: DiscordNotificationAPI, G[
```
・`ConcurrentEffect`は`Sync`を満たすので`ConcurrentEffect`と`Sync`を2つ並べることはできません。
・今回必要なのがnotification類で要求される`Sync`と`merge`で要求される`Concurrent`なので、`Concurrent`を利用すれば要求を満たすことができます。

Type classes in Cats: https://cdn.rawgit.com/tpolecat/cats-infographic/master/cats.svg

Reply:
詳細なクラス図のご紹介ありがとうございました。指摘通り`Concurrent`で対処いたします。